enclave_wrangler/dataset_upload.py

```diff
@@ -34,11 +34,245 @@
 CSET_UPLOAD_REGISTRY_PATH = os.path.join(UPLOADS_DIR, 'cset_upload_registry.csv')
 
 
-def post_to_enclave_and_update_code_sets_csv(input_csv_folder_path) -> pd.DataFrame:
+def post_to_enclave(input_csv_folder_path: str, enclave_api=['old_rid_based', 'default'][1]):
```

Comment:
For this function which accepts a palantir-3-file directory path as its only param, I've left an option to use the old RID based API if they pass `enclave_api='old_rid_based'`. By default, this function will use the new API.

Reply:
Another thing that I could change, since Amin said that these endpoints are literally the same, is to merge my "new API" functions with Stephanie's old, RID based ones. And add  param for choosing which of those APIs to use, defaulting to the new API.